plugin/pom.xml

```diff
@@ -83,9 +83,9 @@
     </dependency>
 
     <dependency>
-      <groupId>io.jenkins.plugins</groupId>
-      <artifactId>commons-text-api</artifactId>
-      <version>1.9-9.v39a_53e2e0343</version>
+      <groupId>org.apache.commons</groupId>
```

Comment:
Maybe leave a TODO comment so #1979 can be reapplied once https://github.com/jenkinsci/commons-lang3-api-plugin/issues/18 is solved properly?